raven/contrib/tornado/__init__.py

```diff
@@ -11,6 +11,7 @@
 
 from tornado import ioloop
 from tornado.httpclient import AsyncHTTPClient, HTTPError
+from tornado import web
```

Comment:
Or uh, `WebHTTPError` would make more sense since the other `HTTPError` is also from tornado. :(